### PR TITLE
okular: add completion

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
This PR is a first PR of in PR series aiming to add simple completions for all KDE and GNOME apps. More advanced completions will be created later.

All such PRs are going to be based on info provided in man pages and not on other sources.

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
This is my first PR, so if you can provide any suggestions, please do it. I wanna conform the best practices used here.

**Comment by akinomyoga** [comment]
Have you checked [CONTRIBUTING](https://github.com/scop/bash-completion/blob/master/CONTRIBUTING.md). Have you considered submitting the completion scripts to the upstream? If yes, could you provide us with the information such as a pointer to the discussion or a copy of it? If not, please consider it. If you have already considered it and decided not to submit it to the upstream, what is the reason?

@scop There are many pull requests submitted without asking it to the upstream projects. Maybe we can include this in the PR template. What do you think?

File: completions/okular [hunk 1]
```diff
@@ -0,0 +1 @@
+complete -W '-p --page --presentation --print --print-and-exit --unique --noraise --find --editor-cmd' okular
```
Comment (akinomyoga) [inline completions/okular:1]: This setting overwrites the existing okular completion by `bash-completion`. `bash-completion` already provides [the completion for `okular` by `_filedir_xspec`](https://github.com/scop/bash-completion/blob/79e9d9392f4f9dbb65177d9554321df5d5b064c5/bash_completion#L2694-L2695), which provides basic completions such as variable name completion, filenames of redirections, [`xspec` for okular](https://github.com/scop/bash-completion/blob/79e9d9392f4f9dbb65177d9554321df5d5b064c5/bash_completion#L2557), etc. These features will be lost by this PR, though it adds a completion for options.

**Review by akinomyoga — CHANGES_REQUESTED** [review]

**Comment by EmilySeville7cfg** [comment]
I think will I contact KDE developers about this completion. Sorry, I thought that I can sent PRs here without asking upstream project maintainers like in Fish shell.

**Comment by scop** [comment]
Right, it's not clear. CONTRIBUTING.md acknowledges that, and tries to elaborate on the tradeoffs and "rules".

**Pull request closed without merging**